src/analysis/classic/warlock/demonology/guide/DoomguardSection.tsx

```diff
@@ -0,0 +1,62 @@
+import { SpellLink } from 'interface';
+import SPELLS from 'common/SPELLS/classic';
+import { GuideProps, Section, SubSection } from 'interface/guide';
+import CombatLogParser from '../CombatLogParser';
+import { FoundationHighlight as HL } from 'interface/guide/foundation/shared';
+
+export function DoomguardSection({ modules, events, info }: GuideProps<typeof CombatLogParser>) {
+  function SnapshotQuality() {
+    if (modules.doomguard.doomguardSummonData.doomguard.summonTimestamp !== 0) {
+      return (
+        <>
+          Snapshot quality:
+          {modules.doomguard.snapshotQualityEntries.map((entry) => {
+            return entry.snapshotSummary;
+          })}
+          {modules.doomguard.snapshotAdvice}
+        </>
+      );
+    } else {
+      return <></>;
+    }
+  }
+
+  return (
+    <Section title="Summon and buff your doomguard">
+      <p>
+        <SpellLink spell={SPELLS.SUMMON_DOOMGUARD} /> is your most powerful damage cooldown, and
+        contributes to a large portion of your DPS. <br />
+        In most of fights, you can use this cooldown only once per fight, due to its 10 mins
+        cooldown, and it lasts for 1:05 mins (45s base duration + 20s thanks to{' '}
+        <SpellLink spell={SPELLS.ANCIENT_GRIMOIRE} /> talent)
+        <br />
+      </p>
+      <p>
+        What makes <SpellLink spell={SPELLS.SUMMON_DOOMGUARD} /> very strong is its{' '}
+        <strong>snapshot mechanism</strong>:<br />
+        When you summon your doomguard, it snapshots spellpower, haste <strong>rating</strong> (such
+        as <SpellLink spell={SPELLS.HURRICANE_BUFF} /> and{' '}
+        <SpellLink spell={SPELLS.ESSENCE_OF_THE_RED} />) and mastery you have when it is summoned,
+        meaning that{' '}
+        <HL>doomguard get benefits of the procs you have at cast for its entire duration</HL>! It is
+        worth nothing that speed cast buffs (like <SpellLink spell={SPELLS.BLOODLUST} />,{' '}
+        <SpellLink spell={SPELLS.HEROISM} /> or <SpellLink spell={SPELLS.POWER_INFUSION} />
+        )are not snapshoted .<br />
```

Comment:
```suggestion
        <SpellLink spell={SPELLS.SUMMON_DOOMGUARD} /> snapshots your stat ratings when it is cast, <strong>including <TooltipElement content={<>Buffs like <SpellLink spell={SPELLS.HURRICANE_BUFF} /> and{' '}
        <SpellLink spell={SPELLS.ESSENCE_OF_THE_RED} /> are the most common</>}>buffs!</TooltipElement></strong>. <HL>This allows your Doomguard to benefit from <em>short buffs</em> for its entire duration.</HL>     
        </p>
        <p><small>Cast speed buffs like <SpellLink spell={SPELLS.BLOODLUST} />,{' '}
        <SpellLink spell={SPELLS.HEROISM} /> or <SpellLink spell={SPELLS.POWER_INFUSION} /> are not snapshotted.</small></p>
```

Reply:
I have adapted a bit here.
The original idea was to make it clear that only haste rating buffs are snapshotted, and not cast speed buffs, because there are often misunderstandings about this.
I'd also prefer to not make Essence of the Red SpellLink part of a tooltip, because you can't hover the link anymore to see what buff it gives to you